inst/rmd/h/navigation-1.1/tabsets.js

```diff
@@ -132,15 +132,6 @@ window.buildTabsets = function(tocID) {
       tabset.rmarkdownStickyTabs();
   }
 
-  // Pandoc 2.8 adds attributes on headers instead of their parent divs, and we
-  // need to move the 'class' attribute from header to div (#1723)
-  $('.tabset').filter(':header').each(function(i, el) {
```

Comment:
Would it have the same effect if we kept this implementation but changed `$('.tabset').filter(':header').each` to `$(':header').each` (or `$(':header:first-child').each`)?

Reply:
Essentially, yes, but a few things to note:

* `$(':header')` is not very performant https://api.jquery.com/header-selector/
* We'd still need to add/remove other attributes besides `class` (to support flexdashboards `data-*` attributes)
* We'd still need to move this logic outside of tabsets.js and into a general dependency of `html_document_base()` (if we've found a relevant pandoc version)